tools/internal_ci/linux/grpc_xds_k8s.sh

```diff
@@ -13,13 +13,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-set -ex -o igncr || set -ex
+set -eo pipefail
 
 # Constants
 readonly GITHUB_REPOSITORY_NAME="grpc"
-# GKE Cluster
-readonly GKE_CLUSTER_NAME="interop-test-psm-sec-v2-us-central1-a"
-readonly GKE_CLUSTER_ZONE="us-central1-a"
+readonly TEST_DRIVER_INSTALL_SCRIPT_URL="https://raw.githubusercontent.com/grpc/grpc/master/tools/internal_ci/linux/grpc_xds_k8s_install_test_driver.sh"
```

Comment:
nit: Maybe we want to add repo owner and branch?

```sh
TEST_DRIVER_INSTALL_SCRIPT_URL="https://raw.githubusercontent.com/${TEST_DRIVER_REPO_OWNER}/grpc/${TEST_DRIVER_BRANCH}/tools/internal_ci/linux/grpc_xds_k8s_install_test_driver.sh"
```

And I wonder if we can reduce the overlap between `TEST_DRIVER_BRANCH`, `TEST_DRIVER_REPO_OWNER`, and `TEST_DRIVER_REPO_URL`.

Reply:
I included the repo owner and branch in the URL.